README.md

```diff
@@ -42,6 +42,7 @@ Samson will:
 * [Extra features](https://github.com/zendesk/samson/blob/master/docs/extra_features.md)
 * [Plugins](https://github.com/zendesk/samson/blob/master/docs/plugins.md)
 * [Statistics](https://github.com/zendesk/samson/blob/master/docs/stats.md)
+* [API](https://github.com/zendesk/samson/blob/master/docs/api.md)
```

Comment:
these should also all be relative

Reply:
Should I change all the other links that I have nothing to do about in my endeavour of improving the doc? I don't mind, but I'm just trying to follow the existing conventions.